crates/holochain_state/src/lib.rs

```diff
@@ -24,10 +24,6 @@
 //! The Query trait allows combining arbitrary database SQL queries with
 //! the scratch space so reads can union across the database and in-memory data.
 
-// TODO - address the underlying issue:
-#![allow(clippy::result_large_err)]
```

Comment:
Fixed this because it's a valid warning. That's a lot of the changes on this PR

Reply:
The problem is put in a box now, got it.